### PR TITLE
added kompot directory

### PR DESCRIPTION
**Pull request by velomanka**
Please add this to master, @casey-collab

### CODE REVIEW

File: russian/kompot/kompot.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Компот забродил и вы тоже. 
+
+1. [И тут вы просыпаетесь](wakeup/wakeup.md)
+2. [Вы продолжаете бродить](walk/walk.md)
```
Comment (casey-collab) [inline russian/kompot/kompot.md]: There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.

File: russian/kompot/kompot.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Компот забродил и вы тоже.
```
Comment (casey-collab) [inline russian/kompot/kompot.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by velomanka** [comment]
"Fixed. Please review this, @casey-collab".

**Comment by casey-collab** [comment]
Looks great! :night_with_stars:

**Pull request merged**